pkg/googlegroups/groupreader.go

```diff
@@ -58,7 +58,13 @@ func (g GroupReader) Descendants(ctx context.Context, groupID string) ([]*groups
 				// Instead, we infer the type from the resource name. e.g. groups have a resource
 				// name of the form `groups/%s` and users of the form 'users/%d'
 				if strings.HasPrefix(m.Member, "users/") {
-					members = append(members, &groupsync.User{ID: m.Member})
+					// m.Id has format of `users/<user-id-numers>`
+					// user's email will be stored in m.PreferredMemberKey
+					// for better user experience, we should user email address instead
```

Comment:
I think we should make this change. Can you create an issue for it so we don't forget to do this?

Reply:
This has been changed.

Previously it's `members = append(members, &groupsync.User{ID: m.Member})`
Now it's `members = append(members, &groupsync.User{ID: m.PreferredMemberKey[0].Id})`

The comment here is explaining why using `PreferredMemberKey.Id` instead of m.Member`